src/fs/host.ts

```diff
@@ -74,10 +74,12 @@ export class IframeHost implements Host {
               "'projects' array should contain at least one item."
             );
           }
-          if (typeof data.projects[0] !== "string") {
-            throw new Error("Expected string");
+          if (typeof data.projects[0] === "string") {
+            resolve({ isDefault: false, main: data.projects[0] });
+          }
+          if (typeof data.projects[0] === "object") {
+            resolve(data.projects[0]);
```

Comment:
Is this basic type checking good enough? Applies to `handleImportProject` below as well.